src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -386,19 +392,11 @@ private void ensureDefaultOdoConfigFileExists() {
     private CustomResourceDefinitionContext toCustomResourceDefinitionContext(org.jboss.tools.intellij.openshift.utils.odo.Service service) {
         String version = service.getApiVersion().substring(service.getApiVersion().indexOf('/') + 1);
         String group = service.getApiVersion().substring(0, service.getApiVersion().indexOf('/'));
-        return new CustomResourceDefinitionContext.Builder()
-                .withName(service.getKind().toLowerCase() + "s." + group)
-                .withGroup(group)
-                .withScope(Scope.NAMESPACED.value())
-                .withKind(service.getKind())
-                .withPlural(service.getKind().toLowerCase() + "s")
-                .withVersion(version)
-                .build();
+        return new CustomResourceDefinitionContext.Builder().withName(service.getKind().toLowerCase() + "s." + group).withGroup(group).withScope(Scope.NAMESPACED.value()).withKind(service.getKind()).withPlural(service.getKind().toLowerCase() + "s").withVersion(version).build();
```

Comment:
```java
.withPlural(service.getKind().toLowerCase() + "s"
```

fabric8 kubernetes-client has a nice utility to turn singular kinds to the plural form:
https://github.com/fabric8io/kubernetes-client/blob/main/kubernetes-model-generator/kubernetes-model-core/src/main/java/io/fabric8/kubernetes/api/Pluralize.java